App/src/App.tsx

```diff
@@ -62,9 +69,34 @@ export function App() {
                     <Route path={BaseRoutes.Register} element={<Register setUser={setUser} />} />
                     <Route
                         path={BaseRoutes.ManageableGroup}
```

Comment:
In addition to having this route, can you also add a button to the home screen to bring the users to the manageable groups page? (As with the routes, keep it hidden unless the user is an admin or group master).

Reply:
sure, was gonna add it with the next pull request with remove from group but I can add it here